app/konnect/advanced-analytics/llm-reporting.md

```diff
@@ -0,0 +1,118 @@
+
+With LLM usage reporting, you can:
+
+* Track token consumption: Monitor the number of tokens processed by the different LLM models you have configured. 
+* Understand costs: Gain visibility into the costs associated with your LLM providers. 
+* Measure Latency: Analyze the latency involved in processing LLM requests. 
+
+To use this feature, navigate to the [Explorer dashboard](https://cloud.konghq.com/us/analytics/explorer) and switch between API usage and LLM usage using the new dataset dropdown. Metrics and groupings will dynamically adjust based on the selected dataset. 
+
+{:.note}
+>**Note**: LLM usage reporting requires at least {{site.base_gateway}} version 3.8. 
+
+
+
+## Metrics
+
+Traffic metrics provide insight into which of your services are being used and how they are responding. Within a single report, you have the flexibility to choose one or multiple metrics from the same category.
+
+| Attribute            | Unit          | Description                                                                                                                                                                                                                                                                           |
+|----------------------|---------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Completion Tokens     | Count         | Completion tokens are any tokens that the model generates in response to an input.       |
+| Prompt Tokens         | Count         | Prompt tokens are the tokens input into the model.            |
+| Total Tokens          | Count         | Sum of all tokens used in a single request to the model. It includes both the tokens in the input (prompt) and the tokens generated by the model (completion).  |
+| Time per Tokens       | Number  | Average time in milliseconds to generate a token. Calculated as LLM latency divided by the number of tokens.                                                       |
+| Costs                 | Cost  | Represents the resulting costs for a request. Final costs = (total number of prompt tokens × input cost per token) + (total number of completion tokens × output cost per token) + (total number of prompt tokens × embedding cost per token). |
+| Response Model        | String        | Represents which AI model was used to process the prompt by the AI provider.   |
+| Request Model         | String        | Represents which AI model was used to process the prompt. |
+| Provider Name         | String        | Represents which AI provider was used to process the prompt.   |
+| Plugin ID             | String        | Represents the UUID of the plugin configuration.          |
```

Comment:
```suggestion
| Plugin ID             | String        | Represents the UUID of the [plugin](/plugins/) configuration.          |
```
Or maybe this should be the link to AI plugins? I think this means just AI plugins, but I'm not sure.

Reply:
Removed the mention of AI